CHANGELOG.asciidoc

```diff
@@ -62,15 +68,11 @@ https://github.com/elastic/beats/compare/1.0.0...master[Check the HEAD diff]
 ////////////////////////////////////////////////////////////
 
 [[release-notes-1.0.1]]
-=== Beats version 1.0.1 https://github.com/elastic/beats/compare/1.0.0...1.0.1[Check 1.0.1 diff]
+=== Beats version 1.0.1
+https://github.com/elastic/beats/compare/1.0.0...1.0.1[Check 1.0.0 diff]
```

Comment:
`To make the link working, it has to be 1.0.0\...1.0.1` But this can also be fixed in https://github.com/elastic/beats/pull/602 afterwards.